vscode_extension/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Ruby IDE features, powered by Sorbet.",
   "author": "Stripe Inc.",
   "license": "Apache-2.0",
-  "version": "0.3.8",
+  "version": "0.3.9",
```

Comment:
This version has already been published, because I tested the non-dry-run version on my branch before putting up the PR.

https://marketplace.visualstudio.com/items?itemName=sorbet.sorbet-vscode-extension